en/grouping.html

```diff
@@ -1312,21 +1312,21 @@ <h2 id="pagination">Pagination</h2>
 reproduce the same result by passing the <em>this</em> continuation along the original select:
 </p>
 <pre>
-select (&hellip;) | [{ 'continuations':['BGAAABEBCA'] }]all(&hellip;);
+select (&hellip;) | { 'continuations':['BGAAABEBCA'] }all(&hellip;)
```

Comment:
Thanks! I was not sure if this is parsed as the annotations, so was not sure whether this can be removed

Reply:
The outer one can, the inner is an array :-)